src/Input/InputParamObjectHandler.php

```
@@ -0,0 +1,206 @@
+
+            throw new ParameterException($message);
+        }
+
+        return $constructorArgs;
+    }
+
+    /**
+     * @param array<string, mixed>    $data
+     * @param ReflectionClass<object> $refClass
+     */
+    public function newInstance(
```

PHPStan (2.1.15)

Method BEAR\Resource\Input\InputParamObjectHandler::newInstance() never returns null so it can be removed from the return type.


(return.unusedType)